app/controllers/admin/countries_controller.rb

```diff
@@ -71,4 +75,13 @@ def set_country
     def country_params
       params.require(:country).permit(:name, :iso, :shp_url, :geojson_url)
     end
+
+  private
+    def sort_column
```

Comment:
For this method you could do the same as sort_direction, and you could get the model name by doing: `controller_name.classify.constantize`